CareKit/CareKit/iOS/Task/View Controllers/OCKSurveyTaskViewController.swift

```diff
@@ -32,31 +32,58 @@
 import CareKitStore
 import CareKitUI
 import ResearchKit
+#if canImport(ResearchKitUI)
+import ResearchKitUI
+#endif
```

Comment:
Would it be possible to change the entire diff to something like

```suggestion
#if canImport(ResearchKitUI)
import ResearchKitUI
public typealias ORKTaskViewControllerFinishReason = ORKTaskFinishReason
#endif
```